src/server/contest/runner.py

```diff
@@ -77,44 +74,43 @@ def check_result(self, input_file):
             # compilation/runtime error
             self.tests.append(result['error'])
 
-    def execute(self, file_path, input_file_path):
+    def execute(self, file_path: Path, input_file_path: Path):
         """Run files on local computer.
         file_path is the name of the file with absolute path """
 
-        # file_path cotains the name of the file with path and extension
-        file_path_without_extension = os.path.splitext(file_path)[0]
-        # contains directory part of file_path
-        file_dir = os.path.dirname(file_path)
-        # name of the file
-        file_name = os.path.basename(file_path)
-        # name of the file without extension
-        file_name_without_extension = file_name.split('.')[0]
+        # # file_path cotains the name of the file with path and extension
```

Comment:
Please explain these changes? It seems you've modified the code AND commented it. Consider removing instead of commenting?

Reply:
That part of the code was intended just for internal testing (even before my changes). I changed the code only to maintain uniformity, but the variables declared there are not used anywhere after that, so I commented those lines out.